src/definitions/modules/flyout.less

```
@@ -0,0 +1,850 @@
+    .ui[class*="seven wide"].right.flyout:not(.fullscreen) {
+      width: @sevenWide;
+    }
+
+    .ui[class*="eight wide"].left.flyout:not(.fullscreen),
+    .ui[class*="eight wide"].right.flyout:not(.fullscreen) {
+      width: @eightWide;
+    }
+
+    .ui[class*="nine wide"].left.flyout:not(.fullscreen),
+    .ui[class*="nine wide"].right.flyout:not(.fullscreen) {
+      width: @nineWide;
+    }
+
+    .ui[class*="ten wide"].left.flyout:not(.fullscreen),
+    .ui[class*="ten wide"].right.flyout:not(.fullscreen) {
+      width: @tenWide;
+    }
+
+    .ui[class*="eleven wide"].left.flyout:not(.fullscreen),
+    .ui[class*="eleven wide"].right.flyout:not(.fullscreen) {
+      width: @elevenWide;
+    }
+
+    .ui[class*="twelve wide"].left.flyout:not(.fullscreen),
+    .ui[class*="twelve wide"].right.flyout:not(.fullscreen) {
+      width: @twelveWide;
+    }
+
+    .ui[class*="thirteen wide"].left.flyout:not(.fullscreen),
+    .ui[class*="thirteen wide"].right.flyout:not(.fullscreen) {
+      width: @thirteenWide;
+    }
+
+    .ui[class*="fourteen wide"].left.flyout:not(.fullscreen),
+    .ui[class*="fourteen wide"].right.flyout:not(.fullscreen) {
+      width: @fourteenWide;
+    }
+
+    .ui[class*="fifteen wide"].left.flyout:not(.fullscreen),
+    .ui[class*="fifteen wide"].right.flyout:not(.fullscreen) {
+      width: @fifteenWide;
+    }
+
+    .ui[class*="sixteen wide"].left.flyout:not(.fullscreen),
+    .ui[class*="sixteen wide"].right.flyout:not(.fullscreen) {
+      width: @sixteenWide;
+    }
+  }
+}
+
+
+
+/*******************************
+          Animations
+*******************************/
+
+& when (@variationFlyoutOverlay) {
+  /*--------------
+      Overlay
+  ---------------*/
+
+  /* Set-up */
+  .ui.overlay.flyout {
+    z-index: @topLayer;
+  }
+
+  /* Initial */
+  & when (@variationFlyoutLeft) {
+    .ui.left.overlay.flyout {
+      transform: translate3d(-100%, 0, 0);
+    }
+  }
+  & when (@variationFlyoutRight) {
+    .ui.right.overlay.flyout {
+      transform: translate3d(100%, 0, 0);
+    }
+  }
+  & when (@variationFlyoutTop) {
+    .ui.top.overlay.flyout {
+      transform: translate3d(0%, -100%, 0);
+    }
+  }
+  & when (@variationFlyoutBottom) {
+    .ui.bottom.overlay.flyout {
+      transform: translate3d(0%, 100%, 0);
+    }
+  }
+
+  /* Animation */
+  .animating.ui.overlay.flyout,
+  .ui.visible.overlay.flyout {
+    transition: transform @duration @easing;
+  }
+
+  /* End - flyout */
+  & when (@variationFlyoutLeft) {
+    .ui.visible.left.overlay.flyout {
+      transform: translate3d(0%, 0, 0);
+    }
+  }
+  & when (@variationFlyoutRight) {
+    .ui.visible.right.overlay.flyout {
+      transform: translate3d(0%, 0, 0);
+    }
+  }
+  & when (@variationFlyoutTop) {
+    .ui.visible.top.overlay.flyout {
+      transform: translate3d(0%, 0, 0);
+    }
+  }
+  & when (@variationFlyoutBottom) {
+    .ui.visible.bottom.overlay.flyout {
+      transform: translate3d(0%, 0, 0);
+    }
+  }
```

This is independent of the positioning as it is always the same transform value, so this can be reduced to
```suggestion
  /* End - flyout */
  .ui.visible.overlay.flyout {
    transform: translate3d(0%, 0, 0);
  }
```

---

Resolved by deb1a561dfc99c26494d0e5c503c1118c7d4ccd3.